concent_api/api_testing_common.py

```diff
@@ -72,6 +74,14 @@ def assert_condition(actual, expected, error_message = None):
         raise TestAssertionException(message)
 
 
+def compare_lists_regardless_of_order(actual: List[str], expected: List[str]) -> bool:
+    assert isinstance(actual, List)
```

Comment:
Should it be `list` ? Isn't `List` only a typing ?

Reply:
done